value/src/main/java/com/google/auto/value/processor/AutoValueProcessor.java

```diff
@@ -585,13 +674,6 @@ private static void determineObjectMethodsToGenerate(
             errors = true;
           }
           toImplement.add(method);
-        } else {
-          // This could reasonably be an error, were it not for an Eclipse bug in
-          // ElementUtils.override that sometimes fails to recognize that one method overrides
-          // another, and therefore leaves us with both an abstract method and the subclass method
-          // that overrides it. This shows up in AutoValueTest.LukesBase for example.
-          errorReporter.reportWarning("@AutoValue classes cannot have abstract methods other than"
```

Comment:
Is there logic to make this check once extensions have had a chance to claim abstract methods? I didn't see it.